### ISSUE TITLE
twisted.logger.PredicateResult is undocumented

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8242]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#8242|
|Type|defect|
|Created|2016-03-21 21:42:55Z|
|Branch|https://github.com/twisted/twisted/tree/predicate-8242|

All its actual _values_ need to be documented as `@cvar`s, because they are presently invisible on https://twistedmatrix.com/documents/16.0.0/api/twisted.logger.PredicateResult.html.

<details><summary>Searchable metadata</summary>

```
trac-id__8242 8242
type__defect defect
reporter__glyph glyph
priority__normal normal
milestone__None None
branch__branches_predicate_8242 branches/predicate-8242
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1458596575912936 1458596575912936
changetime__1458718859417301 1458718859417301
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [47041]) Branching to predicate-8242.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

Not a whole lot going on here, RFR.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @glyph|
|-|-|

Changes looks good.

Please spin the buildbot before merging just to make sure twistechecker is a happy.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph commented|
|-|-|

Buildbots look happy.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set status to `closed`|
|-|-|

(In [47055]) Merge predicate-8242: Document PredicateResult

Author: glyph

Reviewer: adiroiban

Fixes: [#8242](https://github.com/twisted/twisted/issues/8242)

Add docstrings for the values of twisted.logger.PredicateResult.
